README.md

```diff
@@ -80,16 +80,24 @@ publish this contact message privately to some feeds / groups (see e.g.
 
 ### `ssb.friends.isFollowing(opts, cb)` ("async" muxrpc API)
 
-Calls back `true` if `opts.source` follows `opts.dest`, `false` otherwise.
-
+Calls back `true` if `opts.source` follows `opts.dest`, `false` otherwise, where
 `opts.source` and `opts.dest` are strings of SSB Feed IDs.
 
-### `ssb.friends.isBlocking({source, dest}, cb)` ("async" muxrpc API)
+If you pass `opts.details = true`, then the callback will respond with the
+object `{ response, private }`, where `response` is the boolean indicating
+the follow relationship, and `private` is a boolean indicating that the
+relationship was originally encoded in a private (encrypted) message.
 
-Calls back `true` if `opts.source` blocks `opts.dest`, `false` otherwise.
+### `ssb.friends.isBlocking({source, dest}, cb)` ("async" muxrpc API)
```

Comment:
Shouldn't this say `{source, dest, details}` ? Or just `opts` like we seem to do for all the other function docs here.

Reply:
Will fix!